src/Workspaces/CoreTest/Renamer/RenamerTests.VisualBasic.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.UnitTests.Renamer
+{
+    public partial class RenamerTests
+    {
+        [Fact]
+        public Task VisualBasic_TestEmptyDocument()
+        => TestRenameDocumentVisualBasic(
+    "",
+    "",
+    newDocumentName: "NewDocumentName");
```

Comment:
Strange indenting is strange.

Reply:
If the formatter says it's right... /shrug

I'll indent though :)